app/controllers/api/v1/posts_controller.rb

```diff
@@ -15,6 +15,25 @@ def show
       rescue ActiveRecord::RecordNotFound
         render json: { message: 'Post does not exist' }, status: :not_found
       end
+
+      def create
+        user = User.find(params[:user_id])
```

Comment:
```suggestion
        user = User.find(params[:user_id])
```
Move it into a before_action since you're using it in the index method as well.